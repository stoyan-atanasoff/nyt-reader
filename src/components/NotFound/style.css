h1.page-not-found {
  text-align: center;
  font-size: 22px;
  padding-top: 30px;
}
