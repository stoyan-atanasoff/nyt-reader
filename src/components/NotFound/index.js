import React from 'react';
import './style.css';

const NotFoundPage = () => <h1 className='page-not-found'>Page not found!</h1>;

export default NotFoundPage;
