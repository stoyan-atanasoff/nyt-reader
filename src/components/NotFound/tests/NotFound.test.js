import React from 'react';
import NotFound from '../index';
import {render} from "@testing-library/react";

describe('NotFound', () => {
  it('Not Found page should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
        <NotFound />
    );
    expect(firstChild).toMatchSnapshot();
  });
});
