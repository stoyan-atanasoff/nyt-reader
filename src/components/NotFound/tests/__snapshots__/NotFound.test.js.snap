// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound Not Found page should render and match the snapshot 1`] = `
<h1
  class="page-not-found"
>
  Page not found!
</h1>
`;
